app/models/app.rb

```diff
@@ -140,8 +140,16 @@ def validate(record)
   validates :privacy_url,
             uri: true
 
+  validate :functionToCheckAppLaunchFields
```

Comment:
This is minor, but could you stick to the Ruby convention of underscores_in_identifiers? Rather than camelCase.

Reply:
sure no problem. ooo GitHub is letting me do something cool where I can change the code right from here. going to try it let me know what happens.
```suggestion
  validate :check_app_launch
```